### PR TITLE
DDCE-2593 - delete unused urls

### PR DESCRIPTION
**Pull request by frikit**

### CODE REVIEW

File: project/AppDependencies.scala [hunk 1]
```diff
@@ -21,8 +20,8 @@ object AppDependencies {
     "com.github.tomakehurst" % "wiremock-standalone"      % "2.27.2",
```
Comment (frikit) [inline project/AppDependencies.scala:20]: cant update this one compplains on missing logback impl

Comment (frikit) [reply project/AppDependencies.scala]: fixed

**Comment by lmugnai** [comment]
Can we update these?:
org.wiremock:wiremock-standalone:test : 3.0.4                     -> 3.1.0                            
uk.gov.hmrc:play-frontend-hmrc        : 7.19.0-play-28            -> 7.20.0-play-28

**Comment by lmugnai** [comment]
There are some unused vals in FrontendAppConfig that can be deleted.

**Pull request merged**